systems/controllers/controlUtil.cpp

```diff
@@ -111,7 +111,7 @@ void surfaceTangents(const Vector3d & normal, Matrix<double,3,m_surface_tangents
 
 int contactPhi(RigidBodyManipulator* r, SupportStateElement& supp, void *map_ptr, VectorXd &phi, double terrain_height)
 {
-  std::unique_ptr<RigidBody>& b = r->bodies[supp.body_idx];
+  std::shared_ptr<RigidBody>& b = r->bodies[supp.body_idx];

```

Comment:
RigidBodyManipulator bodies changed from unique_ptr to shared_ptr because there are many persistent references to the bodies now.